Process_guide.md

```diff
@@ -207,17 +207,17 @@ It's very important, in order to keep an entire distributed team moving forward,
 
--   Be as atomic as possible. Don't open one ticket for three different, unrelated problems that will likely require three different people working on them (note that JIRA allows for sub-issues that can be assigned to different people, if necessary). Ideally, one ticket is one problem that one person can fix and close. Obviously, sometimes what appears to be one problem on the surface may turn out to be much more complicated
+-   Be as atomic as possible. Don't open one ticket for three different, unrelated problems that will likely require three different people working on them (note that JIRA allows for sub-issues that can be assigned to different people, if necessary). Ideally, one ticket is one problem that one person can fix and close. Obviously, sometimes what appears to be one problem on the surface may turn out to be much more complicated.
 
 -   Contain all relevant information to understand and reproduce the problem without talking directly to the submitter. Whether code, screenshots, attachments, etc. Ideally, the receiver of the ticket may never need to contact you again, and they will be able to fix your problem without further input from you.
 
 # Repository and license management on Github
 
-On Github, we will create teams that correspond to each of the official teams in the KBase org chart. These teams will have ‘admin privileges’ (in Github parlance), which enables their members to create repos owned by that team. As a matter of policy, team members should check with their team leader before creating a new repository. They can always start work if needed on a personal repo and transfer ownership to the team afterwards if an experiment pans out.
+On Github, we will create teams that correspond to each of the official teams in the KBase org chart. These teams will have ‘admin privileges’ (in Github parlance), which enable their members to create repos owned by that team. As a matter of policy, team members should check with their team leader before creating a new repository. They can always start work if needed on a personal repo and transfer ownership to the team afterwards if an experiment pans out.
 
 On Github, all team members can merge PRs, and should consider all their repos to be a common responsibility, engaging with code review on all of them.
 
-If additional areas of work in KBase require teams that cut across the existing Org chart divisions, the leads of areas 3-7 can either make them directly, or ask any of the owners team to make a new one.
+If additional areas of work in KBase require teams that cut across the existing Org chart divisions, the leads of areas 3-7 can either make them directly, or ask any of the owner's team to make a new one.
```

Comment:
This one should be reverted or modified before acceptance.  There is a github "owners" team, which this is referencing.  It is not a grammatical error.  We could say github "owners" team possibly.